features/git-rename-branch/feature_branch/with_remote_origin/with_tracking_branch/with_unpushed_changes.feature

```diff
@@ -0,0 +1,33 @@
+Feature: git rename-branch: errors if renaming a feature branch that has unpushed changes
+
+  As a developer renaming a feature branch that has unpushed changes
+  I should get an error that the given branch is not in sync with its tracking branch
+  So that I don't lose work by accidentally overwriting existing branches.
```

Comment:
Whoops, yes, good catch. Here we can then say what you said in the other comment ("that I know branches must be in sync in order to be renamed").